interop/node-version.json

```diff
@@ -1,19 +1,17 @@
 {
-    "id": "node-js-libp2p-head",
-    "containerImageID": "node-js-libp2p-head",
-    "transports": [
-        "tcp",
-        "ws",
-        {
-            "name": "wss",
-            "onlyDial": true
-        }
-    ],
-    "secureChannels": [
-        "noise"
-    ],
-    "muxers": [
-        "mplex",
-        "yamux"
-    ]
-}
\ No newline at end of file
+  "id": "firefox-js-libp2p-head",
```

Comment:
Why is `firefox-js-libp2p-head` in `interop/node-version.json`?

Reply:
good catch, whoops